### PR TITLE
Cognito import

### PR DESCRIPTION
**Pull request by CedricProfessionnel**
It offer the possibility to import a project existing in aws(only the annotation).
It DON'T :
handle the assets(image, video, audio ...)
export to Aws

It need the pull request : https://github.com/UniversalDataTool/dataset-managers/pull/6 to work

Click on the button Cognito : 
image

Select the project : 
image

and click on take sample from project and all will be download

### CODE REVIEW

**Comment by seveibar** [comment]
glad to see this is coming along :)

File: README.md [hunk 1]
```diff
@@ -127,6 +128,7 @@ Thanks goes to these wonderful people ([emoji key](https://allcontributors.org/d
 
 <!-- markdownlint-enable -->
 <!-- prettier-ignore-end -->
+
```
Comment (seveibar) [inline README.md:131]: this currently breaks the welcome page if introduced, so if we're going to do this reformatting we need to fix the welcome page's parsing of the community updates

Comment (seveibar) [reply README.md:131]: it's not very easy to test

File: src/components/ImportFromCognitoS3Dialog/index.old.js [hunk 2]
```diff
@@ -0,0 +1,452 @@
+// @flow weak
+import React, { useState, useEffect, useRef, useCallback } from "react"
```
Comment (seveibar) [inline src/components/ImportFromCognitoS3Dialog/index.old.js:2]: remove this file, it'll be in the history

File: .github/workflows/test.yml [hunk 3]
```diff
@@ -23,6 +23,11 @@ jobs:
   cypress-run:
     needs: iscodeclean
     runs-on: ubuntu-16.04
+    env:
+      CYPRESS_RECORD_KEY: ${{ secrets.CYPRESS_RECORD_KEY }}
+      CYPRESS_CACHE_FOLDER: "/home/runner/.cache/Cypress"
+      TERM: xterm
+      CI: 1
```
Comment (seveibar) [inline .github/workflows/test.yml:30]: why did you change this? It might be better i'm just not sure why it's better

File: src/components/ImportFromCognitoS3Dialog/index.js [hunk 4]
```diff
@@ -56,7 +34,7 @@ const customStyles = {
 }
 
 const ExpandedRow = ({ data }) => {
```
Comment (seveibar) [inline src/components/ImportFromCognitoS3Dialog/index.js:36]: i would move this to an adjacent file if possible to make the overall number of lines smaller.

**Review by seveibar — CHANGES_REQUESTED** [review]

File: .github/workflows/test.yml (hunk 3 above)
Comment (seveibar) [reply .github/workflows/test.yml:30]: I think you should revert this because it seems to be causing issues by setting protected environment variables on github actions

File: README.md (hunk 1 above)
Comment (CedricProfessionnel) [reply README.md:131]: I will correct the fetch of the starting page

File: src/components/ImportFromCognitoS3Dialog/index.js (hunk 4 above)
Comment (CedricProfessionnel) [reply src/components/ImportFromCognitoS3Dialog/index.js:36]: Which function/var are you refering to ?

File: src/components/StartingPage/index.js [hunk 5]
```diff
@@ -173,15 +173,19 @@ export default ({
...
       const endCU = readme.search("COMMUNITY-UPDATE:END")
-      const communityUpdates = readme
+      var communityUpdates = readme
         .slice(startCU, endCU)
         .split("\n")
         .slice(1, -1)
+      for(var i =communityUpdates.length;i>=0;i--){
+        if(communityUpdates[i]==="")
+          communityUpdates.splice(i,1)
+      }
```
Comment (seveibar) [inline src/components/StartingPage/index.js:187]:
```javascript
const communityUpdates = readme	     
        .slice(startCU, endCU)
        .split("\n")
        .slice(1, -1)
        .filter(line => line.trim() !== "")
```

**Review by seveibar — CHANGES_REQUESTED** [review]
